### PR TITLE
added solution hw-5.3

### PR DESCRIPTION
**Pull request by Serhii-Maksymov**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -9,6 +15,27 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> stringsFromFile = null;
+        try {
+            stringsFromFile = Files.readAllLines(Paths.get(fileName));
+        } catch (IOException e) {
+            e.printStackTrace();
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:22]: don't igrone exceptions

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -9,6 +15,27 @@
...
+        }
+        StringBuilder wordsStartsWithW = new StringBuilder();
+        for (String oneString : stringsFromFile) {
+            oneString = oneString.toLowerCase().replaceAll("\\?|\\!|\\.|\\,|\\;|\\:", "");
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java:26]: simplify the regex

Comment (Serhii-Maksymov) [reply src/main/java/core/basesyntax/FileWork.java]: Ok. Done.

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (Serhii-Maksymov) [reply src/main/java/core/basesyntax/FileWork.java]: I try, Done

**Pull request closed without merging**